src/domain/octree/OctreePacketProcessor.ts

```diff
@@ -70,30 +72,50 @@ class OctreePacketProcessor {
      *  @function OctreePacketProcessor.addedEntity
      *  @returns {Signal}
      */
-    get addedEntity(): Signal {
-        return this.#_addedEntity.signal();
+    get entityData(): Signal {
+        return this.#_entityData.signal();
     }
 
     // Listener
-    // eslint-disable-next-line
-    // @ts-ignore
-    #processPacket = (message: ReceivedMessage, sendingNode: Node | null): void => {// eslint-disable-line
+    #processPacket = (message: ReceivedMessage, sendingNode: Node | null): void => {
         // C++ void OctreePacketProcessor::processPacket(QSharedPointer<ReceivedMessage> message, SharedNodePointer sendingNode)
 
-        const packetType = message.getType();
+        assert(sendingNode !== null, "OctreePacketProcessor.processPacket()", "Sending node is missing.");
 
-        if (packetType === PacketType.OctreeStats) {
-            console.error("OctreePacketProcessor: Packet type not processed: ", packetType);
-            // WEBRTC TODO: Address further C++ code.
-            return;
+        let messageLocal = message;
+
+        const octreePacketType = messageLocal.getType();
+
+        if (octreePacketType === PacketType.OctreeStats) {
+            // WEBRTC TODO: Address further C++ code - process OctreeStats packet.
+
+            console.error("OctreePacketProcessor: Packet type not processed: ", octreePacketType);
+
+            // WEBRTC TODO: Do not hardcode statsMessageLength.
+            const statsMessageLength = 222;
+            // eslint-disable-next-line @typescript-eslint/no-magic-numbers
+            const piggybackBytes = messageLocal.getMessage().byteLength - statsMessageLength;
+
+            if (piggybackBytes > 0) {
+                // eslint-disable-next-line @typescript-eslint/no-magic-numbers
+                const view = new DataView(
+                    messageLocal.getMessage().buffer.slice(messageLocal.getMessage().byteOffset + statsMessageLength)
+                );
+                const packet = NLPacket.fromReceivedPacket(view, piggybackBytes, sendingNode.getPublicSocket());
+                messageLocal = new ReceivedMessage(packet);
+            } else {
+                return;
+            }
         }
 
+        const packetType = messageLocal.getType();
+
         switch (packetType) {
-            case PacketType.EntityData:
-                PacketScribe.EntityData.read();
-                // WEBRTC TODO: Emit data.
-                this.#_addedEntity.emit();
+            case PacketType.EntityData: {
+                const entityDataDetails = PacketScribe.EntityData.read(messageLocal.getMessage());
+                this.#_entityData.emit(entityDataDetails);
                 break;
+            }
```

Comment:
Let's make the console.error() report just EntityErase condition, and explicitly say "... not processed: EntityErase".

Also, a uint16_t needs to be read / skipped over for the EntityQueryInitialResultsComplete case.